### ISSUE TITLE
Error 500, how to debug?

### ISSUE THREAD

**Issue by LubomirKurpel** [Releem/mysqlconfigurer#85]
After running the script I get:

`500 Internal Server Error Oh no! Something bad happened. Error is:Expecting value: line 1 column 2 (char 1)`

In /tmp/.mysqlconfigurer/z_aiops_mysql.cnf. But it does not tell me why it returned error 500, my my.cnf file is in root folder. What can I do? Thanks in advance!

**Comment by drupaladmin** [comment]
Hi, thank you for issue.

We've checked logs and find errors:
SELECT VERSION()[!!] FAIL Execute SQL / return code: 256[!!] You probably did not get enough privileges

It means Releem Agent did not enough privileges to execute queries in MySQL.

Please check ".my.cnf" file and add there root access to MySQL server.

**Comment by LubomirKurpel** [comment]
Hi, thank you for a quick response.

I realised I had my.cnf instead of .my.cnf file so I renamed it, but the creditials inside are right, I double-checked them and logged in with them via phpmyadmin so they should be fine. I tried also with sudo but no luck there.

What else can I check if its a permission problem? Thanks in advance!

**Comment by drupaladmin** [comment]
The same problem.

Full error message you can read at "/tmp/.mysqlconfigurer/mysqltunerreport.json"

Try to run 
1. sudo su, 
2. create .my.cnf at ~/.my.cnf
3. run Releem Agent manually.

**Comment by LubomirKurpel** [comment]
I tried deleting the file and creating with vim, still the same error. Not sure what I am missing.

```
root@vmi404622:/# ls -l .my.cnf
-rw-r--r-- 1 root root 39 Apr  9 19:05 .my.cnf
```

Permissions looks fine to me. I am running Ubuntu 18.04, sudo su does nothing (I am root already).

I have also checked permissions on mysqltuner.pl and its the same:

```
root@vmi404622:/tmp/.mysqlconfigurer# ls -l mysqltuner.pl
-rw-r--r-- 1 root root 231769 Apr  9 19:05 mysqltuner.pl
```

I ran the Releem Agent manually but the response is the same, except for it does not download mysqltuner file on every run.
